compiler/tests/fixtures/compile_errors/return_call_to_fn_with_param_type_mismatch.fe

```diff
@@ -0,0 +1,7 @@
+contract Foo:
+
+    pub def foo(val: address) -> u256:
```

Comment:
This test proves the type checking for the arguments works as expected (trying to pass `100` where an `address` is expected)